src/lib/viewers/media/MediaControls.js

```diff
@@ -671,6 +671,16 @@ class MediaControls extends EventEmitter {
         }
     }
 
+    /**
+     * Prevents hiding of the controls and show media controls.
+     *
+     * @return {void}
+     */
+    disableHideAndShow() {
+        this.preventHiding = true;
```

Comment:
Do you need another function to enable hiding?

Reply:
When user starts to tab around, this user is considered as an accessibility user and hiding is disabled. Enable hiding is not necessary.